packages/server/graphql/types/GitLabIntegration.ts

```diff
@@ -144,42 +147,55 @@ const GitLabIntegration = new GraphQLObjectType<any, GQLContext>({
             projectsFullPaths.add(edge?.node?.fullPath)
           }
         })
-        const projectIssues = [] as ProjectIssueConnection[]
-        const errors = [] as Error[]
-        const hasNextPage = true
-
-        const projectsIssuesPromises = Array.from(projectsFullPaths).map((fullPath) =>
-          manager.getProjectIssues({
+        const projectsIssuesPromises = Array.from(projectsFullPaths).map((fullPath) => {
+          const parsed = args.after && JSON.parse(args.after)
+          const after = (parsed && parsed[fullPath]) ?? ''
+          return manager.getProjectIssues({
             ...args,
-            includeSubepics: true,
-            fullPath
+            fullPath,
+            after
           })
-        )
+        })
+        const projectsIssues = [] as ProjectIssueEdge[]
+        const errors = [] as Error[]
+        let hasNextPage = false
+        const cursors = {} as Cursors
```

Comment:
If the user selects two or more project filters, we want to paginate each project. Meaning we want to show 25 more issues for both projects, not just 25 more issues in total. 

To do that, we need to store the cursor for each selected project filter. In Relay, the cursor needs to be a string, so we're storing the `cursors` object as JSON.